marlowe-dashboard-client/src/Component/ContractPreview/View.purs

```diff
@@ -28,30 +28,30 @@ import Halogen.HTML (a, attr, div, h3, h3_, li, p, p_, slot, text)
 import Halogen.HTML.Events.Extra (onClick_)
 import Halogen.HTML.Properties (title)
 import Halogen.HTML.Properties.ARIA (role)
+import Halogen.Store.Monad (class MonadStore)
 import Humanize (contractIcon)
 import Marlowe.Execution.State (contractName) as Execution
 import Marlowe.Execution.State (currentStep)
 import Marlowe.Extended.Metadata (_contractName, _contractType)
 import Marlowe.Semantics (_rolesCurrency)
 import Page.Contract.Lenses (_marloweParams, _metadata)
 import Page.Dashboard.Types (Action(..), ChildSlots, ContractState)
+import Store as Store
 
 -- This card shows a preview of synced contracts (intended to be used in the dashboard)
 contractPreviewCard
```

Comment:
Main changes in this file:

1. Updates to conform to API changes in Component/Contract/View.purs